### PR TITLE
Commented doctests that were causing slowness at Travis.

### LINKED ISSUE #1037: Some of the eulier doctests are taking a long time
Our Travis CI tests are slowing down.  Travis is still way faster than LGTM but...

Are there any of these doctests that could/should be optimized?  Watch a Travis job run to see which pytest tests are taking a long time to complete.

@brunohadlich

### PR DESCRIPTION
**Pull request by brunohadlich**
Fixes #1037  # This syntax will autoclose 1037 when this PR gets merged.

Tests that were too slow to execute on Project Euler, problems 9, 10, 12 and 14 were commented. #1037

### CODE REVIEW

**Comment by cclauss** [comment]
See the first line that I added above.

File: project_euler/problem_14/sol2.py [hunk 1]
```diff
@@ -47,8 +47,8 @@ def collatz_sequence(n):
 def solution(n):
     """Returns the number under n that generates the longest Collatz sequence.
 
-    >>> solution(1000000)
-    {'counter': 525, 'largest_number': 837799}
+    #>>> solution(1000000)
+    #{'counter': 525, 'largest_number': 837799}
```
Comment (cclauss) [inline project_euler/problem_14/sol2.py]: `#<space>` please on both of these lines
Also, please add a comment on why these lines have been commented out.

Comment (brunohadlich) [reply project_euler/problem_14/sol2.py]: Done.

**Comment by cclauss** [comment]
Can you please try syncing with master or remaking these changes on a fresh branch from master?

**Comment by brunohadlich** [comment]
> Can you please try syncing with master or remaking these changes on a fresh branch from master?

Sure I will do it.

**Pull request merged**